### DISCUSSION TITLE
updateTwitterAuth 函数出错

### DISCUSSION THREAD

**Discussion by Gnoriz-Tea** [HCLonely/auto-task-old#50] [Q&A] [answered]
我试过重新绑定 还是会弹出这个 请问一下如何解决这个问题  感谢大佬
![0](https://user-images.githubusercontent.com/66054917/140042886-32aaa127-5d8e-4882-9b28-2e3806480f24.png)
![1](https://user-images.githubusercontent.com/66054917/140043125-0525dd89-b2b0-41f6-8d9c-656cb2bf0141.png)

**Comment by HCLonely** [comment, marked as answer]
已修复
